iothub/service/src/Feedback/MessageFeedbackProcessorClient.cs

```diff
@@ -190,7 +190,7 @@ private async void OnFeedbackMessageReceivedAsync(AmqpMessage amqpMessage)
 
                         var feedbackBatch = new FeedbackBatch
                         {
-                            EnqueuedOnUtc = (DateTime)amqpMessage.MessageAnnotations.Map[MessageSystemPropertyNames.EnqueuedOn],
+                            EnqueuedOnUtc = (DateTimeOffset)amqpMessage.MessageAnnotations.Map[MessageSystemPropertyNames.EnqueuedOn],
```

Comment:
I tried updating this before, and IIRC it failed. This one we may not be able to change.